target/scripts/check-for-changes.sh

```diff
@@ -174,6 +179,33 @@ function _ssl_changes() {
   # They presently have no special handling other than to trigger a change that will restart Postfix/Dovecot.
 }
 
+function _rspamd_changes() {
+  # RSPAMD_DMS_D='/tmp/docker-mailserver/rspamd'
+  if [[ ${CHANGED} =~ ${RSPAMD_DMS_D}/.* ]]; then
+
+    # "${RSPAMD_DMS_D}/override.d"
+    if [[ ${CHANGED} =~ ${RSPAMD_DMS_OVERRIDE_D}/.* ]]; then
```

Comment:
Regex not necessary?

```suggestion
    if [[ ${CHANGED} == "${RSPAMD_DMS_OVERRIDE_D}/"* ]]; then
```

Also think about the **very** likely case, when someone mounts something like ``/tmp/docker-mailserver/rspamd/overrideXd``, which would also match 😆

There are other places where normal string comparison is done with `=~`, I did choose this just as an example.

Reply:
I think this doesn't work. When you match via `==`, you'd need an asterisk before as well, because `${CHANGED}` is basically `dir/file1 file2 dir2/dir3/file3` and matching `file2` would require `== *file2*`, but with regex, you can just use `=~ file2`, hence the regex IIRC.

Reply:
> Can you elaborate please? I don't see why this would be very likely :)

It is not not. That was just a joke 😆

Reply:
Haha, I see 😆